CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 4.0.0 2022-02-10
```

Comment:
breaking changes are changes that will break existing solutions when they upgrade from 3.x to 4.x without changing the code.
if you're changing response for prior existing methods - this MAY or WILL break, depending on hte changes. So these are most likely "breaking".

Reply:
So please add a section listing all breaking changes